proxy_agent/src/redirector/linux.rs

```
@@ -319,22 +319,26 @@ impl BpfObject {
                         let local_ip: u32 = super::string_to_ip(&local_ip);
                         let value = destination_entry::from_ipv4(local_ip, local_port);
                         match policy_map.insert(key.to_array(), value.to_array(), 0) {
-                            Ok(_) => event_logger::write_event(
-                                LoggerLevel::Info,
-                                format!(
-                                    "policy_map updated for destination: {}:{}",
-                                    ip_to_string(dest_ipv4),
-                                    dest_port
-                                ),
-                                "update_redirect_policy_internal",
-                                "redirector/linux",
-                                logger::AGENT_LOGGER_KEY,
-                            ),
+                            Ok(_) => {
+                                event_logger::write_event(
+                                    LoggerLevel::Info,
+                                    format!(
+                                        "policy_map updated for destination: {}:{}",
+                                        ip_to_string(dest_ipv4),
+                                        dest_port
+                                    ),
+                                    "update_redirect_policy_internal",
+                                    "redirector/linux",
+                                    logger::AGENT_LOGGER_KEY,
+                                );
+                                return true;
+                            }
                             Err(err) => {
                                 logger::write(format!("Failed to insert destination: {}:{} to policy_map with error: {}", ip_to_string(dest_ipv4), dest_port, err));
                             }
                         }
                     }
+                    return true;
```

on error will this still return true?

---

Ignore the error because the redirect policy could be not found, removed before.